IncludeToolbox/Commands/TryAndErrorRemoval_Project.cs

```diff
@@ -87,18 +87,9 @@ static Project GetSelectedCppProject(out string reasonForFailure)
 
         private bool ProcessNextFile()
         {
-            if (projectItems == null)
-                return false;
-
-            for (; fileIdx < projectItems.Count; ++fileIdx)
+            while (projectFiles.Count > 0)
             {
-                var item = projectItems.Item(fileIdx);
-                if (item == null)
-                    continue;
-
-                var projectItem = item.Object as ProjectItem;
-                if (projectItem == null)
-                    continue;
+                ProjectItem projectItem = projectFiles.Dequeue();
```

Comment:
Blah.vcxproj.filters' is already open as a project or a solution and cannot be opened in an editor at this time.

;)